README.md

```diff
@@ -0,0 +1,11 @@
+This repository contains a set of easy to use scripts that helps to provision recommended infrastructure for CKEditor Collaboration Server On-Premises. You can use the script as a reference or set up a production-ready service.
+
+Regardless of the infrastructure provider, we're recommending to use layered architecture for the separation of data from the application layer.
```

Comment:
I would add more information about the `layered architecture`, it can be not too clean for a customers.

Reply:
I've added a link to: https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/architecture.html#overview